vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -291,6 +291,9 @@ and
                 do! setupProjectsAfterSolutionOpen() 
             }
         setupProjectsAfterSolutionOpen() |> Async.StartImmediate
+
+        let theme = package.ComponentModel.DefaultExportProvider.GetExport<ISetThemeColors>().Value
```

Comment:
I tied it to language service because it must run once, and the previous design was a brittle Rube Goldberg machine.